scalastyle-config.xml

```diff
@@ -182,7 +182,7 @@
   </check>
   <check class="org.scalastyle.scalariform.CyclomaticComplexityChecker" enabled="true" level="warning">
     <parameters>
-      <parameter name="maximum"><![CDATA[10]]></parameter>
+      <parameter name="maximum"><![CDATA[12]]></parameter>
```

Comment:
because of https://github.com/target/data-validator/pull/35/files#diff-d67948915fce5238cd2873b8b3361017R27-R63, which I'd love to make less complex